### PR TITLE
Simplified model's Options.add_field() a bit.

### PR DESCRIPTION
**Pull request by Taoup**

### CODE REVIEW

File: django/db/models/options.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import copy
 import inspect
-from bisect import bisect
+import bisect
```
Comment (pope1ni) [inline django/db/models/options.py]: Please move this to the top of the list of imports to fix the error from `isort`.

Comment (Taoup) [reply django/db/models/options.py]: > Please move this to the top of the list of imports to fix the error from `isort`.

ok

**Comment by felixxm** [comment]
@pope1ni PyYAML 5.3 fixed one issue, see #12283.

**Comment by felixxm** [comment]
@Taoup Thanks :+1: .

**Pull request merged**